src/components/content/conditions/patient-conditions.tsx

```diff
@@ -89,10 +91,16 @@ export const PatientConditions = withErrorBoundary(
           updateSorts={updateSorts}
           activeCollection={collection}
           hideAdd={readOnly || collection === "other"}
-          onToggleShowHistoric={() =>
-            updateFilters({ showHistoric: !filters.showHistoric })
-          }
           currentSorts={currentSorts[collection]}
+          filterItems={availableFilters(
+            getUnfilteredCollection(
+              patientConditions,
+              otherConditions,
+              collection
+            )
+          )}
```

Comment:
Also not a blocker, but isn't the whole `getUnfilteredCollection` abstraction just doing a ternary statement? 
```suggestion
          filterItems={availableFilters(
            collection === "patient" ? patientConditions : otherConditions
          )}
```
In which case we could also remove the entire `getFilteredCollection` function at bottom of file as well.